addon/src/services/page-title.ts

```diff
@@ -37,24 +64,31 @@ export default class PageTitleService extends Service {
   };
 
   constructor() {
+    // eslint-disable-next-line prefer-rest-params
     super(...arguments);
     this._validateExistingTitleElement();
 
-    let config = getOwner(this).resolveRegistration('config:environment');
+    const config = getOwner(this).resolveRegistration('config:environment') as {
+      pageTitle?: PageTitleConfig;
```

Comment:
Not sure how to get correct types for owner. Says `resolveRegistration` doesn't exist on it.